CONTRIBUTING.md

```diff
@@ -1 +1,58 @@
-Check [waspc README](waspc) for detailed contributing guidelines.
+# Contributing to Wasp
+
+Wasp's compiler is built with Haskell. As an output, you'll get a web application written in React and NodeJS. So there are three ways for you to contribute: 
+- [Haskell-related](#haskell-related)
+- [Web dev](#web-dev) (React, Node, HTML/CSS, database and so on)
+- [Tutorials or Example apps](#tutorials-or-example-apps)
+- [Documentation](#documentation)
+
+## Before you begin 
+
+- Check the [**Getting Started**](https://wasp-lang.dev/docs) guide to get familiar with Wasp's fundamentals. And ideally, build an app from the [**Pick a Tutorial**](https://wasp-lang.dev/docs/pick-a-tutorial) page
+- Figure out what you'd like to help with. It can be code, documentation, tutorials, etc. More on this is below.
+- Join our Discord [![**Discord**](https://img.shields.io/discord/686873244791210014?label=chat%20on%20discord)](https://discord.gg/rzdnErX) for faster communication and feedback. Also, we'll be glad to help you find the issue you'll enjoy working on, depending on your skill set!
+
+Let's jump right in!
+
+---
+
+## Haskell-related
+
+Wasp compiler is implemented in Haskell, but you will also see a lot of Javascript and other web technologies because the Wasp compiler transpilers Wasp code into them.
+
+You don't have to be an expert in Haskell to contribute or understand the code since we don't use complicated Haskell features much -> most of the code is relatively straightforward, and we are happy to help with the part that is not.
+
+Check the [**Wasp compiler README**](https://github.com/wasp-lang/wasp/blob/main/waspc/README.md) for all the detailed instructions and guides.
+
+[**Haskell-related issues can be found here.**](https://github.com/wasp-lang/wasp/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+label%3Ahaskell) 
+
+Feel free to contact us via Discord to ask for an appropriate issue for yourself. Or create new issues directly to work on them.
+
+---
+
+## Web dev
+
+Wasp is a language for developing full-stack web apps. This means there are plenty of tasks related to web development itself. 
+
+[**Web dev issues can be found here.**](https://github.com/wasp-lang/wasp/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+label%3Awebdev)
+
+---
+
+## Tutorials or Example apps
+
+Another great way to help is to create an app with Wasp! We have an [Examples](https://wasp-lang.dev/docs/examples) section on our website. As well as the [Tutorials](https://wasp-lang.dev/docs/pick-a-tutorial) page. Both those sections can be improved and updated with your projects. 
```

Comment:
The idea was to encourage people to build something with Wasp in the first place. And nothing can be more encouraging than being featured in the docs section of the tool you're using. I'd like to leave the possibility for people to create example apps. And we can decide later in what format it will be. 

If the app is great -- it can be made into a tutorial. If it's not great but good enough -- we can ask a developer to commit to maintaining it. And if the dev won't be doing that -- we can remove it from examples list or figure out, whether someone else from the community would like to update it